manual/sphinx/user_docs/advanced_install.rst

```diff
@@ -263,6 +263,144 @@ Ubgl
 
    ./configure --with-netcdf CXXFLAGS=-DMPICH_IGNORE_CXX_SEEK CFLAGS=-DMPICH_IGNORE_CXX_SEEK --with-pdb=/usr/gapps/pact/new_s/lnx-2.5-ib --with-netcdf=/usr/local/tools/netcdf/netcdf-4.1_c++
 
+Draco
+~~~~~
+.. code-block:: bash
+
+    module load gcc/10 cmake/3.18 openmpi/4
+    PREFIX=$HOME/.local/bout-deps
+    BUILD=/ptmp/$USER/bout-deps
+    BOUT_TOP=$(pwd)
+    MKFLAGS="-j 16"
+    
+    HDF5VER=1.12.0
+    NCVER=4.7.4
+    NCCXXVER=4.3.1
+    FFTWVER=3.3.9
+    SUNVER=5.7.0
+    PETSCVER=3.15.0
+    
+    
+    setup() {
+        mkdir -p $PREFIX
+        mkdir -p $PREFIX/lib
+        if test -e $PREFIX/lib64 ; then
+            if ! test -L $PREFIX/lib64 ; then
+                echo "$PREFIX/lib64 exists and is not a symlink to lib - aborting"
+                exit 1
+            fi
+        else
+            ln -s lib $PREFIX/lib64
+        fi      
+        mkdir -p $BUILD
+    }
+    
+    hdf5() {
+        cd $BUILD
+        wget --delete -c https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.12/hdf5-${HDF5VER}/src/hdf5-${HDF5VER}.tar.bz2
```

Comment:
`--delete` is not an option in the version of `wget` on my machine (`GNU Wget 1.21.1 built on linux-gnu.`). Maybe `--unlink`? Or is this a different version of `wget` on this machine?